lib/chore/queues/filesystem/consumer.rb

```diff
@@ -99,7 +117,13 @@ def consume
           Chore.logger.info "Starting consuming file system queue #{@queue_name} in #{self.class.queue_dir(queue_name)}"
           while running?
             begin
-              #TODO move expired job files to new directory?
+              # Move expired job files to new directory (so long as enough time has
+              # passed since we last did this check)
+              if !@last_cleaned_at || (Time.now - @last_cleaned_at).to_i >= MINIMUM_EXPIRATION_CHECK_INTERVAL
```

Comment:
was it intended to have this `MINIMUM_` get overridden?  just an assumption i made based on naming of the constant.  not something i view as a blocker fer sher.

Reply:
Good call, I'll rename this variable.  I could imagine some need for this to be configurable, but until that need is actually there, I would just leave this a constant.

Reply:
Actually, I think the name may be more accurate as-is (I'll spend a few minutes thinking aout it).  The code doesn't guarantee 1s intervals -- it just guarantees that there's at least 1s between checks.

Reply:
Thought about it more and decided to rename it, but leave in the comments indicating that this is more of a minimum value